### DISCUSSION TITLE
One git submodule per rust crate modules?

### DISCUSSION THREAD

**Discussion by yvan-sraka** [massalabs/massa#1913] [Ideas]
When the project will be a bit more stabilized it would be nice to have this kind of splitting :)

**Comment by adrien-zinger** [comment]
Without going too far, we can just do an architecture pass in order to spot linked libs between wasm-simulation and massa

**Comment by AureliaDolo** [comment]
@damip that's what we were talking about downstairs, I convert that to a discussion
